joint_constraints:
  arm_lift_joint:
    position: 0.0
    tolerance_above: 1000.0
    tolerance_below: 1000.0
    weight: 0.5
  arm_flex_joint:
    position: 0.0
    tolerance_above: 1000.0
    tolerance_below: 1000.0
    weight: 0.5
  arm_roll_joint:
    position: 0.0
    tolerance_above: 0.1
    tolerance_below: 0.1
    weight: 1.0
  wrist_flex_joint:
    position: 0.0
    tolerance_above: 1000.0
    tolerance_below: 1000.0
    weight: 0.5
  wrist_roll_joint:
    position: 0.0
    tolerance_above: 1000.0
    tolerance_below: 1000.0
    weight: 0.5
  odom_x:
    position: 0.0
    tolerance_above: 1000.0
    tolerance_below: 1000.0
    weight: 0.5
  odom_y:
    position: 0.0
    tolerance_above: 1000.0
    tolerance_below: 1000.0
    weight: 0.5
  odom_t:
    position: 0.0
    tolerance_above: 1000.0
    tolerance_below: 1000.0
    weight: 0.5
